### PR TITLE
Added validation, note style, edit label to media source editor container

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: src/core/views/media-view.js [hunk 1]
```diff
@@ -47,13 +55,24 @@ define( [ "ui/page-element", "ui/logo-spinner" ], function( PageElement, LogoSpi
...
-        showError( true, "Your URL must not be blank:" );
+        subtitle.className += " form-error";
+        urlTextbox.className += " form-error";
+        showError( true, "Not a valid URL. Use http://..." );
       }
     }
+    
+    function testUrl(url) {
+      var test = /(http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/
+      console.log(url.match(test));
```
Comment (jbuck) [inline src/core/views/media-view.js:73]: nuke it

File: src/core/views/media-view.js [hunk 2]
```diff
@@ -13,27 +13,35 @@ define( [ "ui/page-element", "ui/logo-spinner" ], function( PageElement, LogoSpi
     _propertiesElement.className = "butter-media-properties";
     _propertiesElement.setAttribute( "data-butter-exclude", true );
 
+    var editMessage = document.createElement( "p");
+    editMessage.className = "edit-message";
+    editMessage.innerHTML = "Edit...";
```
Comment (jbuck) [inline src/core/views/media-view.js]: I liked the "Edit source" better in the UI concept, personally

File: src/core/views/media-view.js [hunk 3]
```diff
@@ -47,13 +55,24 @@ define( [ "ui/page-element", "ui/logo-spinner" ], function( PageElement, LogoSpi
...
       else{
-        showError( true, "Your URL must not be blank:" );
+        subtitle.className += " form-error";
+        urlTextbox.className += " form-error";
+        showError( true, "Not a valid URL. Use http://..." );
       }
     }
+    
+    function testUrl(url) {
+      var test = /(http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/
```
Comment (jbuck) [inline src/core/views/media-view.js:72]: Not a huge fan of this regex, but I guess it is needed.

Comment (k88hudson) [inline src/core/views/media-view.js:72]: I think we need it for now, since not including http will cause an error

**Pull request merged**